docs/modules/general/pages/bootstrapping.adoc

```diff
@@ -0,0 +1,567 @@
+
+Module imports:: 
+* CommonModule, BrowserModule and BrowserAnimationsModule Angular modules used for adding functionality to the module.
+* PortalCoreModule is defined to allow usage of OneCX components and services.
+* TranslateModule is defined to allow translations using translation keys within the module.
+// TODO: Add link to auth docs
+* AngularAuthModule is defined to use OneCX authorization mechanisms.
+* RouterModule is defined for routing to feature modules within the exposed module.
+
+Entrypoint component::
+// TODO: Add link to createAppEntrypoint
+AppEntrypoint is a standard Angular component, which has a `<router-outlet>` element in its template. The `createAppEntrypoint` registers AppEntrypointComponent in the Custom Elements Registry, so anytime '<example-component>' is rendered, AppEntrypointComponent should be instantiated.
++
+The third parameter, being the module's Injector, is very important. This injector will be used by the instances of AppEntrypointComponent rendered using Web Components technology, meaning that each instance will have everything related to the module already set up. That also means the `<router-outlet>` will be using routes defined for the module.
++
+// TODO: Add link to createAppEntrypoint
+// TODO: Link to events topic
+[[module-rotuer-connection]]The `createAppEntrypoint` method is also responsible for connecting the module's router to Shell's router. Every time the URL of the browser changes, Shell is going to publish a new message, via EventsTopic, with information about the new URL. The `createAppEntrypoint` method subscribes to the EventsTopic and updates the router state accordingly to the received information.
++
+For technologies other than Angular, it is recommended to:
++
+--
+* register Custom Element in the Custom Elements Registry.
+* provide dependencies to registered Custom Element according to the module.
+* listen to EventsTopic data changes and update the state of the module's routing.
+--
+
+Routes matching::
+Each route defined will load a feature module whenever it is activated. Because Webcomponent expose method causes multiple routers to exist at the same time (Shell has its own router and every module or Remote Component displayed at a single point in time can have their own), an adjustment to the routes definition has to be made.
++
+The idea of routing in this example is the following:
++
+--
+* User enters 'shell_url/workspace_name/example_base_path' URL - FeatureModule is used.
+* User enters 'shell_url/workspace_name/example_base_path/tracking' URL - TrackingModule is used.
+--
++
+With the following URL parts meaning:
++
+--
+* shell_url - the Shell Application deployment URL, e.g. `localhost:4200/shell.`
+* workspace_name - name of the accessed Workspace, e.g. `admin`.
+* example_base_path - base path of the example Microfrontend (configured via OneCX Core Applications), e.g. `example`.
```

Comment:
`shell_url`
`workspace_name`
`example_base_path`
or use any other kind of formatting